lib/functions/rootfs/distro-agnostic.sh

```diff
@@ -288,6 +288,13 @@ install_common() {
 It is not too late to `unset KERNELSOURCE` here and avoid kernel install.
 PRE_INSTALL_KERNEL_DEBS
 
+	# install board support packages
```

Comment:
What would changing location help?

Reply:
We want to move postinstall scripts to /etc/kernel/... scripts. These scripts will be installed with BSP packages.
If we install kernel before BSP, then /etc/vmlinux symlink will not be created.

Reply:
This part needs a broader discussion and testing on all variants (sadly our testing capacity is limited). Too afraid to change this handling, while half of the PR is just fine.